src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -154,43 +154,55 @@ public with sharing class PMT_PaymentWizard_CTRL {
 
     /*******************************************************************************************************
     * @description decimal for amount already written off
-    */ 
+    */
     private Decimal amountWrittenOff { get; set; }
 
 
     /*******************************************************************************************************
     * @description Integer for count of invalid payment records
-    */ 
+    */
     private Integer invalidPaymentRecordCount { get; set; }
 
     /*******************************************************************************************************
     * @description Boolean to indicate if the wizard page should be enabled
-    */ 
+    */
     public Boolean isWizardEnabled { get; set; }
-    
+
     /*******************************************************************************************************
     * @description gets the payment related settings
     * @return npe01__Contacts_and_Orgs_Settings__c the settings object
-    */ 
+    */
     public npe01__Contacts_and_Orgs_Settings__c getPaymentsSettings(){
         if (paymentsSettings == null) {
            paymentsSettings = UTIL_CustomSettingsFacade.getContactsSettings();
         }
-        
+
         return paymentsSettings;
     }
-  
+
+    /*******************************************************************************************************
+    * @description Returns true if Payments enabled is true on paymentsSettings
+    * @return Boolean
+    */
+    public Boolean getIsPaymentsEnabled() {
+        if (getPaymentsSettings() == null) {
+            return false;
+        }
+        return getPaymentsSettings().npe01__Payments_Enabled__c;
```

Comment:
Normally I'd say this is fine, however Checkbox fields on a Custom Settings object is the one exception where the field value can actually be null (normally checkbox fields are _always_ either true or false). So, to avoid an error evaluating something like `!isPaymentsEnabled` in the VF page, I recommend modifying this statement as follows. This way the method explicitly returns true only if the checkbox value is true and returns false otherwise.
```suggestion
        return getPaymentsSettings().npe01__Payments_Enabled__c == true;
```